src/controllers/OrdersController.php

```diff
@@ -1151,26 +1151,26 @@ private function _getTransactionsWIthLevelsTableArray($transactions, $level = 0)
                     'id' => $transaction->id,
                     'level' => $level,
                     'type' => [
-                        'label' => Plugin::t(StringHelper::toTitleCase($transaction->type)),
+                        'label' => htmlspecialchars(Plugin::t(StringHelper::toTitleCase($transaction->type))),
                         'level' => $level,
                     ],
                     'status' => [
                         'key' => $transaction->status,
-                        'label' => Plugin::t(StringHelper::toTitleCase($transaction->status))
+                        'label' => htmlspecialchars(Plugin::t(StringHelper::toTitleCase($transaction->status)))
                     ],
                     'amount' => Plugin::getInstance()->getPaymentCurrencies()->convert($transaction->amount, $transaction->currency) . ' <small class="light">(' . $transaction->currency . ')</small>',
                     'paymentAmount' => Plugin::getInstance()->getPaymentCurrencies()->convert($transaction->paymentAmount, $transaction->paymentCurrency) . ' <small class="light">(' . $transaction->currency . ')</small>',
-                    'gateway' => $transaction->gateway->name ?? Plugin::t('Missing Gateway'),
+                    'gateway' => htmlspecialchars($transaction->gateway->name ?? Plugin::t('Missing Gateway')),
```

Comment:
these probably aren't necessary? all coming back from a gateway and i'm assuming a user never has a change to change them?